### PR TITLE
feat: Object validation

### LINKED ISSUE #303: Objects not validating properly anymore
Target-csv is failing to validate objects coming from tap-postgres now. It looks like they are coming out as a string, but they aren't being converted properly.

### PR DESCRIPTION
**Pull request by sebastianswms**
Modifies the schema for jsonb and json objects because using an empty schema was not previously function for all taps (ex: tap-csv).

Closes #303

### CODE REVIEW

**Review by edgarrmondragon — APPROVED** [review]
Did some testing and it does seem to work as expected!

File: tap_postgres/client.py [hunk 1]
```diff
@@ -45,7 +48,26 @@ def patched_conform(
     """
     if isinstance(elem, datetime.date):
         return elem.isoformat()
-    return unpatched_conform(elem=elem, property_schema=property_schema)
+    if isinstance(elem, (datetime.datetime, pendulum.DateTime)):
+        return singer_sdk.helpers._typing.to_json_compatible(elem)
+    if isinstance(elem, datetime.timedelta):
```
Comment (visch) [inline tap_postgres/client.py:53]: Could we call the super here instead of copying the code?

**Review by visch — CHANGES_REQUESTED** [review]

File: tap_postgres/client.py (hunk 1 above)
Comment (sebastianswms) [reply tap_postgres/client.py]: Comments added explaining why this isn't possible.

**Comment by visch** [comment]
> Did some testing against target-postgres and it does seem to work as expected!

Thank you :D

**Review by visch — APPROVED** [review]

**Pull request merged**